docs/ai-consumers/tokens.md

```diff
@@ -24,21 +24,21 @@ However, to understand the different network tokens, you can follow principles l
 * Mainnet tokens could have a monetary value.
 * Both tokens can be used to add extra utility to the software.
 
-We only use Kovan and Ropsten Testnet AGI and the current beta stage. 
+We only use Kovan and Ropsten Testnet AGIX and the current beta stage. 
 
-To learn more about  Mainnet AGI Token (to be used in a later stage) refer to this link [here](https://etherscan.io/address/0x8eb24319393716668d768dcec29356ae9cffe285) with the address `0x8eb24319393716668d768dcec29356ae9cffe285`.
+To learn more about  Mainnet AGIX Token (to be used in a later stage) refer to this link [here](https://etherscan.io/address/0x8eb24319393716668d768dcec29356ae9cffe285) with the address `0x8eb24319393716668d768dcec29356ae9cffe285`.
 
 <div class="callout callout--warning">
-    <p><strong>NOTE</strong> Currently, AGI token is traded on some exchanges, and also the speculative secondary trading is considered against the ethics of AGI token and SingularityNET project. Therefore, it is not recommended to exchange trading in any manner.</p>
+    <p><strong>NOTE</strong> Currently, AGIX token is traded on some exchanges, and also the speculative secondary trading is considered against the ethics of AGIX token and SingularityNET project. Therefore, it is not recommended to exchange trading in any manner.</p>
 </div>
 
 
-## AGI Faucet
-To understand how automated faucet is used in distribution of Kovan and Ropsten Testnet AGI Tokens, click on the link [here](http://faucet.singularitynet.io). Use GitHub account to login and type in the Ethereum address to receive the AGI token. Ensure this address belongs Kovan Ethereum Address. 
+## AGIX Faucet
+To understand how automated faucet is used in distribution of Kovan and Ropsten Testnet AGIX Tokens, click on the link [here](http://faucet.singularitynet.io). Use GitHub account to login and type in the Ethereum address to receive the AGIX token. Ensure this address belongs Kovan Ethereum Address. 
 
-**Note**: You can request for 10 AGI token every 24 hours.
+**Note**: You can request for 10 AGIX token every 24 hours.
 
-In order to add the Kovan Testnet AGI, you may need the following details:
-* Symbol: AGI
+In order to add the Kovan Testnet AGIX, you may need the following details:
+* Symbol: AGIX
```

Comment:
We need to change the addresses here

Reply:
@anandrgitnirman  Please have a look here...